payments.yaml

```diff
@@ -1,3 +1,6 @@
 product_type:
   'cic-report' : 'CIC report'
   'late-filing-penalty' : 'Late filing penalty'
+  'certificate'                 : 'Certificate of Incorporation'
+  'certificate-same-day'        : 'Certificate of Incorporation same day'
+  'certificate-additional-copy' : 'Additional copy of Certificate of Incorporation'
```

Comment:
It occurs to me I am not sure about this point: are we adding a single product type to this file to cover all certificates, or a product type for each different product code (=cost/cost code)?

Reply:
I have discussed this with Vaughan, and confirmed that these are the correct values to be added, as specified in the specification for the paymentDetails resource.